application/core/LSUploadHelper.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/*
+ * LimeSurvey
+ * Copyright (C) 2020 The LimeSurvey Project Team / Carsten Schmitz
+ * All rights reserved.
+ * License: GNU/GPL License v2 or later, see LICENSE.php
+ * LimeSurvey is free software. This version may have been modified pursuant
+ * to the GNU General Public License, and as distributed it includes or
+ * is derivative of works licensed under the GNU General Public License or
+ * other free or open source software licenses.
+ * See COPYRIGHT.php for copyright notices and details.
+ * 
+ */
+
+class LSUploadHelper
+{
+    /**
+     * Check uploaded file size
+     * 
+     * @param string $sFileName the name of the posted file
+     * @param mixed $customMaxSize maximum file upload size
+     * 
+     * @throws Exception if the file is too large or no file is found.
+     */
+    public static function checkUploadedFileSize($sFileName, $customMaxSize = null)
```

Comment:
No static functions, please! They cannot be mocked easily.

Reply:
OK, will take into account next time.
Can we just leave it like this for now?

Reply:
No? Don't add technical debt in new code. :cry:

Reply:
Done! Debt paid :)